projects/packages/search/src/dashboard/components/pages/dashboard-page.jsx

```diff
@@ -82,6 +84,8 @@ export default function DashboardPage( { isLoading = false } ) {
 						supportsInstantSearch={ supportsInstantSearch }
 						supportsOnlyClassicSearch={ supportsOnlyClassicSearch }
 					/>
+					<FirstRunSection isVisible={ false } />
+					<PlanUsageSection isVisible={ false } />
```

Comment:
This also needs to be gated by the new pricing feature flag —  `isNewPricing`.

Reply:
Hey Jason. The plan is definitely to move to the `isNewPricing` value once the UI is fully are tested but in the meantime, I find it very useful to pull up both sections as needed when manually testing. I'll create a separate bookkeeping Issue on the board to make sure this isn't overlooked though.

Added in #26693.